recipes/stateless-service/08-pipeline-run.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: pipeline-run
+spec:
+  serviceAccountName: pipeline
+  pipelineRef:
+    name: pipeline-build-project
+  resources:
+    - name: src
+      resourceRef:
+        name: git-project
+  workspaces:
+    - name: local-maven-repo
+      persistentVolumeClaim:
+        claimName: maven-repo-pvc
```

Comment:
Are we using it to store our local maven artifacts or?
Maybe we can have an optional setting to configure a maven mirror to speed up the process?